src/state/migrations/index.ts

```diff
@@ -13,4 +13,5 @@ export const migrations = {
   5: clearNfts,
   6: clearAssets,
   7: clearPortfolio,
+  8: clearOpportunities,
```

Comment:
we can delete the comment above since we are using length of this

Reply:
b627e535f8e1619852fd8bbe8bd8124571765d1d